smooth-app-bar-layout/src/main/java/me/henrytao/smoothappbarlayout/SmoothAppBarLayout.java

```diff
@@ -151,6 +151,7 @@ public int getCurrentOffset() {
 
   public void setScrollTargetCallback(ScrollTargetCallback scrollTargetCallback) {
     mScrollTargetCallback = scrollTargetCallback;
+    setBehaviorScrollTargetCallback();
```

Comment:
@henrytao-me I've added you to my repo an a contributor. Feel free to change whatever you want. Seems like I was trying to fix something that wasn't broken. A quick question: what's the purpose of setScrollTargetCallback method in SmoothAppBarLayout? Am I missing something or in your original implementation it does nothing but storing a reference to the passed callback which is never used?